README.rst

```diff
@@ -159,10 +159,8 @@ Rather, you can just import ``mayavi.mlab`` and pass it to any figure list that
 
 Installation for developers
 ---------------------------
-
-Once these are installed,
-to install from github, just ``git clone https://github.com/jmfranck/pyspecdata.git`` then move to the directory where setup.py lives,
-and do
+To install from github, just ``git clone https://github.com/jmfranck/pyspecdata.git``. Then switch over to the anaconda prompt and move to the directory where setup.py lives (ex. ``git/pyspecdata/``),
```

Comment:
what about making sure `conda info` matches, though?

Reply:
README.rst is for installing pyspecdata. I've added the conda info in the step where we install anaconda in the wiki (https://avhe-jmfranck-node1.syr.edu/wiki/index.php/Important_Links#GVim_and_PySpecData). The installation of pyspecdata assumes this is already done properly, but will add a note

Reply:
but `conda info` is relevant to installing pyspecdata as a developer -- ostensibly if you're doing that, you have git bash installed

Reply:
it should be possible to just refer the wiki to the readme -- say do the developer install

Reply:
wiki already points to this guy for installing pyspecdata so I think we are good?